lib/solarwinds_apm/opentelemetry/otlp_processor.rb

```diff
@@ -8,39 +8,46 @@
 
 module SolarWindsAPM
   module OpenTelemetry
-    # reference: OpenTelemetry::SDK::Trace::SpanProcessor; inheritance: SolarWindsProcessor
-    class OTLPProcessor < SolarWindsProcessor
-      attr_accessor :description
```

Comment:
Is there still a need for `solarwinds_processor.rb` then, or can we remove it?

Reply:
No, I am thinking to remove everything once it's ready for pre-release, but I can remove the file here as well.

Reply:
Would be great to remove files as we go to confirm working order... and of course a final sweep after all PRs get merged into pure ruby branch, before merging to main.

Reply:
Removed